lib/components/settings/display/display_settings_screen.dart

```diff
@@ -108,3 +126,36 @@ class DisplaySettingsScreen extends StatelessWidget {
     );
   }
 }
+
+class FontRadioDialogTile extends StatelessWidget {
```

Comment:
```suggestion
class _FontRadioDialogTile extends StatelessWidget {
```
Let's make this private since we will only need it here